f8a_worker/workers/githuber.py

```diff
@@ -52,13 +53,26 @@ def _get_repo_stats(self, repo):
         """Collect various repository properties."""
         try:
             if repo.get('contributors_url', ''):
-                contributors = get_response(repo.get('contributors_url', ''), self._headers)
+                contributors_url = repo.get('contributors_url', '')
+                page_count = 1
+                contributors = []
+
+                while True:
+                    url = contributors_url + "?per_page=100&page=" + str(page_count)
+                    contributors_batch = get_gh_contributors(url, self._headers)
+
+                    if len(contributors_batch) > 0:
+                        page_count += 1
+                        contributors.extend(contributors_batch)
+                        time.sleep(1)
+                    else:
+                        break
             else:
```

Comment:
Instead of constructing logic for pagination, follow the Github Api doc. It provides a way to retrieve the total contributors from the `Link` response header value.

https://developer.github.com/v3/guides/traversing-with-pagination/#basics-of-pagination

Response:
```
HTTP/1.1 200 OK
date: Thu, 30 Jul 2020 12:23:26 GMT
content-type: application/json; charset=utf-8
server: GitHub.com
status: 200 OK
...
link: <https://api.github.com/repositories/218118349/contributors?per_page=1&page=2>; rel="next", <https://api.github.com/repositories/218118349/contributors?per_page=1&page=4>; rel="last"
access-control-expose-headers: ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, Deprecation, Sunset
...
```

Here is a pythonic way to retrieve using requests.

```
import requests
from urllib.parse import urlparse, parse_qs

rsp = requests.head('https://api.github.com/repos/quay/container-security-operator/contributors?per_page=1')
contributors_count = parse_qs(urlparse(rsp.links['last']['url']).query)['page']
```